Added Object level permissions to Profile, Dataset and Point Collection admin

goyal1092 opened the pull request:
* Initial implementation of object level permissions
* Added Django gaurdian and configs
* Added a choice field in Profile, Dataset and PointCollection and added
migration
* Changed admin pages accordingly

@adieyal Please review

adieyal commented:
![](https://github.trello.services/images/mini-trello-icon.png) [Ensure object-level permissions for datasets and point collections](https://trello.com/c/CE4BCW7i/145-ensure-object-level-permissions-for-datasets-and-point-collections)

goyal1092 commented:
@adieyal Updated PR. Please review.

* I have rebased this PR via staging so we can delete branch model_refactor.

Updates: 
* Field named `permission_type` for `Profile, Dataset & Point collection`.
* Fixed bugs in Filtering indicator via geography & dataset
* Custom UI for group permissions that works according to the `permission_type field`.

adieyal commented:
Will do, thanks

On Wed, Apr 22, 2020 at 11:58 AM Gaurav Goyal <notifications@github.com>
wrote:

> @adieyal <https://github.com/adieyal> Updated PR. Please review.
>
>    - I have rebased this PR via staging so we can delete branch
>    model_refactor.
>
> Updates:
>
>    - Field named permission_type for Profile, Dataset & Point collection.
>    - Fixed bugs in Filtering indicator via geography & dataset
>    - Custom UI for group permissions that works according to the permission_type
>    field.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/OpenUpSA/wazimap-ng/pull/49#issuecomment-617679180>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AADGXMQ3LXFQ3OBYT3RHPJTRN25UNANCNFSM4MKSTBNA>
> .
>


-- 
Adi Eyal
Director
OpenUp
Promoting informed decision-making

phone: +27 78 014 2469
skype: adieyalcas
linkedin: http://za.linkedin.com/pub/dir/Adi/Eyal
web: http://www.openup.org.za
twitter: @soapsudtycoon

Follow us on twitter at @OpenUpSA
Sign up for Naked Data, our weekly round-up of what's happening
locally and internationally in data journalism and data visualisation.
Register here: https://openup.org.za/nakeddata.html
Also check out our data storytelling short courses:
https://openup.org.za/trainup/
